File: 3069-distribute-elements-into-two-arrays-i/3069-distribute-elements-into-two-arrays-i.cpp
class Solution {
public:
    vector<int> resultArray(vector<int>& nums) {
        vector<int>ans1;
        vector<int>ans2;
        ans1.push_back(nums[0]);
        ans2.push_back(nums[1]);
        for(int i=2;i<nums.size();i++){
        if(ans1.back()>ans2.back()){
            ans1.push_back(nums[i]);
        }else{
            ans2.push_back(nums[i]);
            }
        }
       
        vector<int>result=ans1;
        for(auto ans:ans2){
            result.push_back(ans);
        }
        return result;

    }
};

File: 3069-distribute-elements-into-two-arrays-i/3069-distribute-elements-into-two-arrays-i.py
class Solution:
    def resultArray(self, nums: List[int]) -> List[int]:
        ans1=[nums[0]]
        ans2=[nums[1]]
        for i in range(2,len(nums)):
            if ans1[-1] > ans2[-1]:
                ans1.append(nums[i])
            else:
                ans2.append(nums[i])    
        return ans1 + ans2